templates/blog/post_list.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <h2>All post:</h2>
```

Comment:
The <h2> tag should be placed inside the <body> tag, not inside the <head> tag. Consider moving it to the appropriate section.